example/rust/Cargo.toml

```diff
@@ -17,7 +17,9 @@ path = "src/docs_examples/docs_examples.rs"
 
 [dependencies]
 futures = "0.3.30"
-momento = { version = "0.39.0" }
+momento = { version = "0.39.0", path = "../../sdk" }
```

Comment:
I assume this and the example are there for development and shouldn't be merged.

Reply:
I'll merge as is and revert in the other pr.

Reply:
correct, i just wanted you to be able to see what the example code looks like and make sure you think it's sane